addons/ieds/functions/fnc_createIEDObject.sqf

```diff
@@ -0,0 +1,71 @@
+
+#include "script_component.hpp"
+
+#define LAND_IEDS ["ACE_IEDLandBig_Range", "ACE_IEDLandSmall_Range"]
+#define URBAN_IEDS ["ACE_IEDUrbanBig_Range", "ACE_IEDUrbanSmall_Range"]
+
+params ["_logic"];
+
+if (isNull _logic) exitwith {};
+
+private _typeOfIED = _logic getvariable ["typeOfIED", 0];
+private _sizeOfIED = _logic getvariable ["sizeOfIED", 0];
+private _heightOfIED = _logic getvariable ["heightOfIED", 0];
+
+_heightOfIED = switch (_heightOfIED) do {
+    case 1: {-0.01};
+    case 2: {-0.05};
+    case 3: {-0.075};
+    case 4: {-0.25};
+    default {_heightOfIED};
+};
+
+private _iedClass = switch (_typeOfIED) do {
+    case 0: { LAND_IEDS select _sizeOfIED };
+    case 1: { URBAN_IEDS select _sizeOfIED };
+};
+
+private _iedCreated = _iedClass createVehicle [0,0,0];
+_iedCreated setPos [getPos _Logic select 0, getPos _Logic select 1, (getPos _Logic select 2) + _heightOfIED];
+
+if (_logic getvariable ["iedActivationType", 0] == 0) then {
+    private _mine = createMine [_iedClass, getPos _iedCreated, [], 0];
+    _mine setDir ((getDir _iedCreated) + 90);
+    _mine setPos _iedCreated;
+    _iedCreated setvariable [QGVAR(pressurePlate), _mine];
+    hideObjectGlobal _mine;
```

Comment:
This only works on the server
https://community.bistudio.com/wiki/hideObjectGlobal
But it can be called by a zeus dedicated client via the module.